cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2309,6 +2309,33 @@
   </property>
 
 
+  <!-- Runtime Monitor Configuration -->
+
+  <property>
+    <name>runtime.monitor.polltime.ms</name>
+    <value>2000</value>
+    <description>
+      Polling time in millis to poll updates from a runtime
+    </description>
+  </property>
+
+  <property>
+    <name>runtime.monitor.batch.limit</name>
+    <value>1000</value>
+    <description>
+      Number of events to fetch from a runtime in each poll call.
+    </description>
+  </property>
+
+  <property>
+    <name>runtime.monitor.topics</name>
+    <value>program.status.record.event.topic</value>
+    <description>
+      A comma-separated list of topic configs to be monitored by runtime monitor.
```

Comment:
You should support syntax as described in the `messaging.system.topics` property.